### DISCUSSION TITLE
Thoughts on University engagement (supporting Student events, providing curriculum templates)

### DISCUSSION THREAD

**Discussion by Willmish** [Green-Software-Foundation/discussions#72] [Community Working Group]
During this week's Community WG call while discussing https://github.com/Green-Software-Foundation/community-wg/issues/83 , a lot of interesting and important issues came up mostly focusing on the issue of how we can collaborate with academic institutions and Student bodies at universities.

Key points:
- If GSF ends up supporting ClimateHack 2023 organised by UCL's AI Society, can we reuse the materials/methods of collaboration for future university partnerships and encourage them - e.g. we can point other universities/student societies asking for help for an example collaboration
- Still not sure what material we can point Universities/lectures to for Green Software content.

@NAMRATA-WOKE @aecurrie @tmcclell please share your insights/topics of discussion too!

**Comment by tholewis** [comment]
I am in the middle of building out curriculum/training for Green Software Foundations. It is a 3-part series:

1. Fundamentals
2. Using Open Source Tools
3. Green Software and AI

It is based off of the existing content from GSF. Although it does contain some Microsoft tools, they can be easily switched out (I'm trying to make it as agnostic as possible and just using Microsoft tools as an example - this content is not a sales tool). Probably end of November is when it will be ready and I am glad to share it broadly in whatever way the GSF would like.

If this University engagement motion becomes a project, I am glad to help support it as I've worked with Learning institutions a lot as well as building content for them.

**Comment by russelltrow** [comment]
Adding the previous discussion for reference: https://github.com/orgs/Green-Software-Foundation/discussions/20

**Comment by tmcclell** [comment]
Proposal to create a project with Thomas/Pindy as Leads, bringing initial proposals to the Community working group. Think about how it might link to Champions program. @adamj89  to be the PM. Working group approved.

**Comment by anewmandev** [comment]
I think this is a really good idea. 

I spoke very briefly to @adamj89 about this as it was a suggestion that came in from our Brighton community after our first meetup in September and it really resonated with me.  

I have spoken to a few faculty staff across computing and business and also to the career depts across University of Sussex and University of Brighton. So far I've been met with a positive initial response but nobody has yet got me in to speak to anyone. I will let you know if I do have any luck. I am taking the polite but persistent door knock approach :) 

My hope is to get a chance to go up at the end of a computing lecture or a careers event or a student community or something and either deliver a full 'Introducing Green Software' talk or even just an informal chat/intro. 

I'd be interested in chatting to anybody who is passionate about this as I really think educatoin, University in particular, is a perfect place to introduce this conversation and there are many ways this can be done. 

Glad to see this discussion :)
